packages/simm-helpers/src/convert-fullwidthhiragana-to-halfwidthkatakana.ts

```diff
@@ -0,0 +1,112 @@
+const FIRST_FULL_WIDTH_HIRAGANA = "ぁ";
```

Comment:
Đối lại tên file: `fullwidth-hiragana-to-halfwidth-katakana.ts`, Tương tự cho phần tests